insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/app_layer/status/GetFirmwareVersionsMessage.kt

```diff
@@ -0,0 +1,31 @@
+package info.nightscout.androidaps.plugins.pump.insight.app_layer.status
+
+import info.nightscout.androidaps.plugins.pump.insight.app_layer.AppLayerMessage
+import info.nightscout.androidaps.plugins.pump.insight.app_layer.Service
+import info.nightscout.androidaps.plugins.pump.insight.descriptors.FirmwareVersions
+import info.nightscout.androidaps.plugins.pump.insight.descriptors.MessagePriority
+import info.nightscout.androidaps.plugins.pump.insight.utils.ByteBuf
+
+class GetFirmwareVersionsMessage : AppLayerMessage(MessagePriority.NORMAL, false, false, Service.STATUS) {
+
+    var firmwareVersions: FirmwareVersions? = null
+        private set
+
+    override fun parse(byteBuf: ByteBuf?) {
+        firmwareVersions = FirmwareVersions()
+        firmwareVersions?.let {
```

Comment:
here I had to manage a several null together (several files in status package with the same situation)
I'm not sure if it's the best way to manage this situation (I used `.let` for `firmwareVersions` (var property of the class) and `.run` for `byteBuf` inside)
what is the best practice in this situation ? (current proposal, 2 let with different iterators, other ?)